server/routerlicious/packages/lambdas/src/deli/checkpointContext.ts

```diff
@@ -108,14 +109,19 @@ export class CheckpointContext {
 		}
 
 		let updateP: Promise<void>;
+        let isLocal = false;
```

Comment:
Nit: is this equivalent to:

`let isLocal = this.localCheckpointEnabled && (checkpoint.reason !== CheckpointReason.NoClients)`